src/java.desktop/share/classes/sun/awt/image/ImageRepresentation.java

```diff
@@ -109,7 +109,7 @@ public synchronized void reconstruct(int flags) {
             src.checkSecurity(null, false);
         }
         int missinginfo = flags & ~availinfo;
-        if ((availinfo & ImageObserver.ERROR) == 0 && missinginfo != 0) {
+        if ((availinfo & (ImageObserver.ERROR |  ImageObserver.FRAMEBITS) ) == 0 && missinginfo != 0) {
```

Comment:
```suggestion
        if ((availinfo & (ImageObserver.ERROR | ImageObserver.FRAMEBITS)) == 0
            && missinginfo != 0) {
```
The space between the closing parentheses is redundant as well as the extra space after `|` operator. The line could be wrapped to fit into 80-column limit.